visualization/app/codeCharta/ui/copyToClipboardButton/util/getFilenamesWithHighestMetric.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { NodeType } from "../../../codeCharta.model"
+import { FileToValue, getFilenamesWithHighestMetrics, updateAttributeMap } from "./getFilenamesWithHighestMetrics"
+
+let MAP: Map<string, FileToValue[]>
+
+beforeEach(() => {
+	MAP = new Map<string, FileToValue[]>([
+		[
+			"mcc",
+			[
+				{ name: "file2", value: 500 },
+				{ name: "file3", value: 400 },
+				{ name: "file1", value: 300 },
+				{ name: "file1", value: 200 },
+				{ name: "file1", value: 100 },
+				{ name: "file1", value: 50 },
+				{ name: "file1", value: 20 },
+				{ name: "file1", value: 15 },
+				{ name: "file1", value: 10 },
+				{ name: "file1", value: 5 }
+			]
+		]
+	])
+})
+
+describe("getFilenamesWithHighestMetrics", () => {
+	it("should give map which has exactly two keys (mcc, rloc)", () => {
+		const resultMap = getFilenamesWithHighestMetrics(BIG_NODE_WITH_TWO_ATTRIBUTES)
+		const mccResults = resultMap.get("mcc")
+		const rlocResults = resultMap.get("rloc")
+
+		expect(resultMap.size).toBe(2)
+		expect(mccResults).toBeTruthy()
+		expect(rlocResults).toBeTruthy()
+	})
```

Comment:
[Optional] add new lines after each `it`-block

Reply:
Resolved with commit [caf8910](https://github.com/MaibornWolff/codecharta/pull/2942/commits/caf8910d32f50acb77cc810979421c23c6c2b353)